hbase-server/src/main/java/org/apache/hadoop/hbase/mob/MobFileCache.java

```diff
@@ -325,4 +326,8 @@ public void printStatistics() {
     lastEvictedFileCount += evicted;
   }
 
+  private long hashFileName(String fileName) {
+    return Hashing.murmur3_32_fixed().hashString(fileName, java.nio.charset.StandardCharsets.UTF_8)
```

Comment:
Better use murmur3_128 and finall use asLong to reduce conflicts? We could use long in IdLock.
And please add comments to say why we use this function.

Reply:
Thanks @Apache9 , good suggestions. PR is updated.